x-pack/plugins/upgrade_assistant/public/application/components/coming_soon_prompt.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiEmptyPrompt, EuiPageContent, EuiLink } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useAppContext } from '../app_context';
+
+export const ComingSoonPrompt: React.FunctionComponent = () => {
+  const { kibanaVersionInfo, docLinks } = useAppContext();
+  const { nextMajor, currentMajor } = kibanaVersionInfo;
+  const { ELASTIC_WEBSITE_URL } = docLinks;
+
+  return (
+    <EuiPageContent>
+      <EuiEmptyPrompt
+        iconType="wrench"
+        data-test-subj="comingSoonPrompt"
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.upgradeAssistant.emptyPrompt.title"
+              defaultMessage="{uaVersion} Upgrade Assistant"
+              values={{ uaVersion: `${nextMajor}.0` }}
+            />
+          </h2>
+        }
+        body={
+          <>
+            <p>
+              <FormattedMessage
+                id="xpack.upgradeAssistant.emptyPrompt.upgradeAssistantDescription"
+                defaultMessage="The Upgrade Assistant identifies deprecated settings in your cluster and helps you
+                resolve issues before you upgrade. Check back here when it's time to upgrade to Elasticsearch {nextMajor}."
```

Comment:
Would it make sense to say "_when it's time to upgrade your stack to..._"?

Reply:
Good point. I changed this to "`when it's time to upgrade your Elastic Stack to XX`".

Reply:
If we want this message to match how we're starting to talk about releases in blogs, maybe it should be "upgrade to Elastic X.XX."

Reply:
Latest: `The Upgrade Assistant identifies deprecated settings in your cluster and helps you
                resolve issues before you upgrade. Check back here when it's time to upgrade to Elastic {nextMajor}.`